apollo-ios-codegen/Sources/GraphQLCompiler/JavaScript/auto_rollup.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
+set -e
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 output_file="$SCRIPT_DIR/../ApolloCodegenFrontendBundle.swift"
-$( cd "$SCRIPT_DIR" && rollup -c )
+cd "$SCRIPT_DIR" && npm run build
```

Comment:
Use the package version of `rollup` instead of needing a global installation of it.